lib/ledger_sync/ledgers/xero/request.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Ledgers
+    module Xero
+      class Request < Ledgers::Request
```

Comment:
Should this be in Core? Anything Xero-specific here?

Reply:
I think this could be moved to core (OAuth).